moveit_planners/stomp/stomp_moveit/package.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<package>
+  <name>stomp_moveit</name>
+  <version>0.0.0</version>
+  <description>The stomp_moveit package</description>
+
+  <maintainer email="jrgnichodevel@gmail.com">Jorge Nicho</maintainer>
+  <license>Apache 2.0</license>
```

Comment:
@jrgnicho most of this pull-request can be improved by external contributors,
but the question whether the code can be BSD-3 licensed can only be answered/discussed by someone involved with SwRI afaics. While nothing is wrong with Apache 2.0 in general, it is preferable to have all of MoveIt under the same license, i.e. BSD-3.

The same is true for the missing author declarations @davetcoleman pointed out.

Could you please take the time to fix these organizational things?
I would really like to see stomp merged upstream and we definitely want to avoid license/copyright problems haunting us in the future..